sdk/communication/communication-call-automation/src/models/models.ts

```diff
@@ -142,3 +142,14 @@ export type RecordingFormat = "mp3" | "mp4" | "wav";
 
 /** The storage type of a call recording. */
 export type RecordingStorage = "acs" | "blobStorage";
+
+/** Channel affinity for a participant */
+export interface ChannelAffinity {
+  /** Channel number to which bitstream from a particular participant will be written. */
+  channel?: number;
+  /**
+   * The identifier for the participant whose bitstream will be written to the channel
+   * represented by the channel number.
+   */
+  participant: CommunicationIdentifier;
```

Comment:
Would targetParticipant make sense here? If it does, please change to it to align our agreement on the naming. If not please ignore.

Reply:
updated to targetParticipant